app/src/main/res/drawable/day_button.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:state_checked="true"
+        android:drawable="@drawable/day_button_selected" />
+
+    <item
+        android:drawable="@drawable/day_button_unselected" />
+
+</selector>
```

Comment:
I've created a new kind of button that automatically changes the background of the button when checked. This is the code for which drawable to display. Each button sets the background of day_button.xml.